client/jetpack-cloud/sections/partner-portal/payment-method-actions/index.tsx

```diff
@@ -33,7 +33,7 @@ const PaymentMethodActions: FunctionComponent< Props > = ( { card } ) => {
 
 	const handleDelete = useCallback( () => {
 		closeDialog();
-		reduxDispatch( deleteStoredCard( card, nextPrimaryPaymentMethod.id ) )
+		reduxDispatch( deleteStoredCard( card, nextPrimaryPaymentMethod?.id ) )
```

Comment:
This is something unrelated to this PR, but I noticed it's broken and decided to fix it here.